src/client/components/TlockEntry.tsx

```diff
@@ -64,7 +64,43 @@ export const TlockEntry = (props: TlockEntryProps) => {
                         />
                     )}</TableCell>
                 </TableRow>
+                <TableRow>
+                    <TableCell>Plaintext</TableCell>
+                    <TableCell>
+                        {entry.plaintext.length === 0
+                            ? ""
+                            : entry.uploadType === "text"
+                                ? Buffer.from(entry.plaintext, "base64").toString("utf-8")
+                                : <Button onClick={download}>Download File</Button>
```

Comment:
Why not allow to download still encrypted files?

Reply:
well you just copy the armor. I guess that could be downloadable too, but I stuck it in a textarea to make it easily copypastable